Providers.asciidoc

```diff
@@ -0,0 +1,84 @@
+== Providers
+
+This chapter describes how to use and configure the providers included
+with Atomic App. It gives you a short overview of all available
+providers and how to use them.
+
+Not all Atomic Apps must support each provider: one Atomic App may
+only include deployment information for OpenShift3 while another does
+support OpenShift3 and Kubernets.
+
+=== List of providers
+
+Atomic App 0.0.2 includes three providers: 
+
+* Docker, which is the most simple one
+* Kubernets
+* OpenShift 3
+
+All providers assume that you install and run the Atomic App on a host
+that is part of the backend cluster or runs docker directly. By now we
+do not support remote deployments. (see also )
+
+=== Choosing a provider
+
+While deploying and Atomic App you can choose one of the providers by
+setting it in `answers.conf`:
+
+```
+{
+    "general": {
+            "provider": "openshift"
+    }
+}
+```
+
+Providers may need additional configuration.
+
```

Comment:
Could you add a reference to Docker provider? Even as a placeholder would be better than leaving it out altogether.